GDJS/scripts/install-spine.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const path = require('path');
+
+const originalSpinePath = path.join('node_modules/pixi-spine/dist', 'pixi-spine.js');
```

Comment:
Parse originalSpinePath from pixi-spine package.json
see prop: `extensionConfig.bundle`
Also use `require.resolve("pixi-spine/package.json")` to find path to package